sdk/core/Azure.Core/src/Internal/AzureCorePipelineEnumerator.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.ClientModel.Core;
+using System.Threading.Tasks;
+
+namespace Azure.Core.Pipeline
+{
+    internal class AzureCorePipelineEnumerator : PipelineEnumerator
```

Comment:
Can we add `Azure` prefix to all the types that are derived from `System.Net.ClientModel` type? It is so much easier to distinguish between them.

Reply:
@AlexanderSher, I like this idea, but before we take steps in that direction, I want to make sure I understand your suggestion.  Can you list a few other types you'd like to see with the Azure prefix to illustrate the idea a bit more?